### ISSUE TITLE
Search returns top matching result, not page of matching results

### ISSUE THREAD

**Issue by BWPanda** [backdrop-ops/api.backdropcms.org#46]
Go to https://api.backdropcms.org/ and enter 'config' into the search field on the right. A list of matching results is displayed below the search field, but you want to view a page showing all/more matches, so you just press Enter without clicking on any of them.

You expect to be taken to https://api.backdropcms.org/search/node/config

You actually get taken to https://api.backdropcms.org/api/backdrop/core%21includes%21config.inc/function/config/1

**Comment by docwilmot** [comment]
yes, always noticed that. If you do "config*" you do get those results.

**Comment by BWPanda** [comment]
Thanks for the tip @docwilmot! I had no idea you could use wildcards like that. Good workaround, but I think this still needs addressing as it's not user-friendly to have to do that all the time...

**Comment by BWPanda** [comment]
It seems this only happens when the top matching result is either an exact match, or the beginning of a match. E.g. searching for `config` takes you to https://api.backdropcms.org/api/backdrop/core%21includes%21config.inc/function/config/1 (exact match), and `hook_` takes you to https://api.backdropcms.org/api/backdrop/core%21modules%21system%21system.api.php/function/hook_init/1 (beginning of a match).

But non-perfect matches work as expected, they take you to the search results page. E.g. `file` takes you to https://api.backdropcms.org/search/node/file (top result is `File`, but case-sensitivity means it's not a perfect match), and `test` takes you to https://api.backdropcms.org/search/node/test (no near matches).
